card/mod/admin/set/self/admin.rb

```diff
@@ -1,6 +1,10 @@
+# collect arrays of the form
+# [task symbol, { execute_policy: block, stats_policy: block }]
+basket :tasks
+
 event :admin_tasks, :initialize, on: :update do
```

Comment:
it sure doesn't seem like rubocop is measuring view/event complexity.  that would probably knock down wikirate's scores quite a bit (and wagn some, too)